Sources/MediaCategoryCells/MediaCollectionViewCell.swift

```diff
@@ -132,3 +132,21 @@ class MediaCollectionViewCell: BaseCollectionViewCell {
         newLabel.isHidden = true
     }
 }
+
+// MARK: - Private helpers
+
+private extension MediaCollectionViewCell {
+    private func name(of artist: VLCMLArtist?) -> String {
+        guard let artist = artist else {
+            return NSLocalizedString("UNKNOWN_ARTIST", comment: "")
+        }
+
+        if artist.identifier() == UnknownArtistID {
+            return NSLocalizedString("UNKNOWN_ARTIST", comment: "")
```

Comment:
I realize that this might be considered as duplication and can be put with the guard. I decided mostly for readability to have a separate condition.

Reply:
yeah that's fine :)